PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!-- 
+Thank you for submitting a pull request! Please add some info (if applicable) to give us some context on the PR.
+
+We will review the PR as soon as possible, leave feedback, add a tag, etc. and let you know what's going on.
+
+Cheers!
+
+The App Center team -->
+
+Things to consider before you submit the PR:
+
+* [x] Has `CHANGELOG.md` been updated?
```

Comment:
This should not be checked by default, it's a real task list generated by github.

Reply:
Other check boxes must have a space : `[ ]`